nextflow.config

```
@@ -155,8 +155,11 @@ params {
     krona_taxonomy_directory   = null
 
     // profile standardisation
-    run_profile_standardisation = false
-    generate_biom_output        = false
+    run_profile_standardisation             = false
+    taxpasta_add_taxonomy                   = false
+    taxpasta_add_samplesheet                = false
+    taxpasta_standardisation_format         = 'tsv'
+    generate_biom_output                    = false
```

```suggestion
    standardisation_motus_generatebiom                    = false
```

Needs to be changed in pipeline code